pkg/cluster/k8sres_test.go

```diff
@@ -451,3 +452,64 @@ func TestSecretVolume(t *testing.T) {
 		}
 	}
 }
+
+func TestTLSConfiguration(t *testing.T) {
```

Comment:
hm, can you rename it to `TestTLS`. Configuration sounds like you're testing parameters from the config map / CRD config